cmd/dep/testdata/harness_tests/init/glide/transitive/trans-trans-conflict/initial/glide.yaml

```diff
@@ -0,0 +1,6 @@
+package: .
+import:
+- package: github.com/chriswhelix/deptestglide1
+  version: gt100
+- package: github.com/chriswhelix/deptestglide2
+  version: eq081
```

Comment:
I'll leave it to @sdboyer to say how "preferred versions" should work. I don't know enough about them to comment. 😀